tests/random-data-set.bats

```diff
@@ -101,6 +113,27 @@
   [ "$status" -eq 0 ]
 }
 
+@test "Marker genes: Add second dataset for deletion tests" {
+  cp $testsDir/marker-genes/TEST-EXP1.clusters.tsv $testsDir/marker-genes/TEST-EXP2.clusters.tsv
+  cp $testsDir/marker-genes/TEST-EXP1.marker_genes_9.tsv $testsDir/marker-genes/TEST-EXP2.marker_genes_9.tsv
+  cp $testsDir/marker-genes/TEST-EXP1.marker_genes_10.tsv $testsDir/marker-genes/TEST-EXP2.marker_genes_10.tsv
+  cp $testsDir/marker-genes/TEST-EXP1.marker_genes_11.tsv $testsDir/marker-genes/TEST-EXP2.marker_genes_11.tsv
+  export EXP_ID=TEST-EXP2
+  run load_db_scxa_marker_genes.sh
+  echo "output = ${output}"
+  [ "$status" -eq 0 ]
+}
+
+@test "Marker genes: Delete rows for experiment" {
+  countBefore=$(echo "SELECT COUNT(*) FROM scxa_marker_genes" | psql $dbConnection | awk 'NR==3')
+  export EXP_ID=TEST-EXP2
+  run delete_db_scxa_marker_genes.sh
+  echo "output = ${output}"
+  [ "$status" -eq 0 ]
+  countAfter=$(echo "SELECT COUNT(*) FROM scxa_marker_genes" | psql $dbConnection | awk 'NR==3')
+  [ $(( countBefore - countAfter )) == 274 ]
```

Comment:
Is 274 difference in counts is expected for all experiments ?

Reply:
It is for the two example experiments (the same but just duplicated with different accessions) that are loaded in the tests (which is controlled within the repo).